Homework2/DAGs/myDAG.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+from datetime import datetime, timedelta
+
+import requests
+import alpha_vantage
+
+from alpha_vantage.timeseries import TimeSeries
+ts = TimeSeries(key='YADS9AQEC5BOFNJK',output_format='pandas')
+# Get json object with the intraday data and another with  the call's metadata
+def Get_stocks():
```

Comment:
Python is case-sensitive: the function name `Get_stocks()` should be the same as what you pass in `t1`. Suggetsed to have them both called `get_stocks()`